packages/@sanity/portable-text-editor/src/editor/PortableTextEditor.tsx

```diff
@@ -255,9 +256,11 @@ export class PortableTextEditor extends React.Component<
     }
     // Sync value from props, but not when we are responding to incoming patches
     // (if this is the case, we sync the value after the incoming patches has been processed - see createWithPatches plugin)
+    const isPristineEditor =
+      !prevProps.value && this.slateInstance.children === this.state.initialSlateEditorValue
     if (
       this.props.value !== prevProps.value &&
-      (!prevProps.value || this.readOnly || !this.props.incomingPatches$)
```

Comment:
This logic was wrong, because prevProps.value could be `undefined` when someone empties the editor.

We want this to only be true on the initial load when in a collaborative setting.